.github/actions/build-image/action.yaml

```
@@ -23,13 +23,13 @@ runs:
   using: "composite"
   steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
```

```suggestion
        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b // v2
```

just having the hash makes it a bit unclear what version is usesd. dependabot will also adds this comment as far as I know.

---

done